### ISSUE TITLE
Question about Training Data Used in Comparison with ShAPO

### ISSUE THREAD

**Issue by kentar0kozai** [zubair-irshad/shapo#23]
Hello,    

First of all, I am deeply impressed by your research.   
I have a question regarding the content of your paper.  

You mentioned that ShAPO is trained using only simulation data, but did the other methods in the comparison also use only simulation data for training?  
Or did the other methods include real-world data in their training as well?  

I look forward to your reply.  
Best regards,

**Comment by kentar0kozai** [comment]
Sorry, I overlooked the description. So it is fine-tuned with real data.

Thank you.

**Comment by zubair-irshad** [comment]
Yes, feel free to checkout out follow-up work, [FSD (ICRA'24)](https://fsd6d.github.io/) that only trains on synthetic data and uses self-supervised losses from real data for fine-tuning (i.e. no 3D real world labels were used in FSD). Hope it helps!

**Comment by kentar0kozai** [comment]
In ShAPO, a small amount of real annotated label data was necessary, but is it correct to understand that in FSD, real annotated labels are no longer required?

Thanks for your reply!

**Comment by zubair-irshad** [comment]
Yes your understanding is correct regarding ShAAPO. For FSD, I should clarify we need 2D masks in FSD along with RGB-D only during training but no 3D mesh models or 3D pose annotations are required for real-world data to train FSD.

**Comment by kentar0kozai** [comment]
I would like to read more about FSD as well.
I will continue to support your work in the future!
